packages/web/src/Components/Select/Select.test.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from 'react';
+import * as TestRenderer from 'react-test-renderer';
+import { render, fireEvent, within } from '@testing-library/react';
+import { ISelectProps, Select } from './Select';
+
+const getProps = (props: Partial<ISelectProps> = {}) => {
+  const {
+    disabled,
+    helpText,
+    id = 'Select',
+    label,
+    options = ['Option1', 'Option2'],
+    placeholder = 'Placeholder',
+    state,
+  } = props;
+
+  return { state, id, placeholder, options, disabled, helpText, label };
+};
+
+describe('Select component', () => {
```

Comment:
senti falta de um teste que garanta o acesso correto do valor selecionado pela usuária.
ex: `should have the correct value` -> seleciona alguma opção -> expect(valor do select).toEqual('blabla')
A ideia aqui é testar a logica do handleChange, mais precisamente, garantir que essa função esta sendo chamada no `onChange`